src/Apps/CollectorProfile/Routes/Saves2/Components/SelectListsForArtworkModal/useUpdateCollectionsForArtwork.ts

```diff
@@ -32,13 +32,13 @@ export const useUpdateCollectionsForArtwork = (artworkID: string) => {
             ... on ArtworksCollectionsBatchUpdateSuccess {
               addedToCollections {
                 internalID
-                artworksCount
                 default
+                ...SavesItem_item
```

Comment:
**question (non-blocking):** To confirm, this solves the problem, because the mutation response now matches the shape of the original query for the rail, correct?

https://github.com/artsy/force/blob/529d18e72071a2e0d8cc3fbe5c875005d7ab5588/src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx#L79-L102